pkg/controller/cache/cache_controller.go

```diff
@@ -99,6 +97,10 @@ func (r *ReconcileCache) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcile.Result{}, err
 	}
 
+	if instance.Spec.AdminAuth != nil {
```

Comment:
Would you like to remove this section from CR if it present and update a CR?

Reply:
Is there any advantage of this approach if the struct is always ignored?

Reply:
Seems to be no value for us, probably b customers will have better and clear view.

Reply:
Actually, it's better for us as well as it will stop the repeated log calls on each reconcile event without requiring #872